### ISSUE TITLE
Installation error on windows

### ISSUE THREAD

**Issue by peerbolte** [FWeinb/electron-screenshot-service#25]
Is the module suppose to work on windows?
Running `npm install electron-screenshot-service` on Windows 10 produces the following error:

```
15231 error code ELIFECYCLE
15232 error electron-screenshot-service@3.2.0 postinstall: `cd ./electron-service; npm install .`
15232 error Exit status 1
15233 error Failed at the electron-screenshot-service@3.2.0 postinstall script 'cd ./electron-service; npm install .'.
15233 error Make sure you have the latest version of node.js and npm installed.
15233 error If you do, this is most likely a problem with the electron-screenshot-service package,
15233 error not with npm itself.
15233 error Tell the author that this fails on your system:
15233 error     cd ./electron-service; npm install .
```

Maybe Windows doesn't like the postinstall script format? 

node v6.3.0
npm v3.10.3
Windows 10

**Comment by FWeinb** [comment]
Windows doesn't seam to like `;` for chaining commands. Will try with `&&`.

**Comment by FWeinb** [comment]
I just published `3.2.1` could you give that a try.

**Comment by peerbolte** [comment]
I installed `3.2.2`successfully on windows. Thanks!
